lib/Froxlor/Settings/Store.php

```diff
@@ -290,6 +290,28 @@ public static function storeSettingIpAddress($fieldname, $fielddata, $newfieldva
 
 	public static function storeSettingMysqlAccessHost($fieldname, $fielddata, $newfieldvalue)
 	{
+        $ips = $newfieldvalue;
+	    //Convert cidr to netmask for mysql, if needed be
+        if(strpos($ips, ',') !== false) {
+            $ips = explode(',', $ips);
+        }
+        if(is_array($ips) && count($ips) > 0) {
+            $newfieldvalue = [];
+            foreach ($ips as $ip) {
+                $org_ip = $ip;
```

Comment:
I would suggest checking for ipv6 address here and not performing the `cidr2NetmaskAddr` validate etc. on it as it's not valid for MySQL to specify ipv6-cidr's

Reply:
I must disagree. Checks are done before hand (`\Froxlor\Validate\Check::checkMysqlAccessHost`) and if we're not validating here but in the save method, what good does that do? It just shifts the problem into a new class instead.

Reply:
Yes, this needs to be done in `Validate\Check::checkMysqlAccessHost()`.

If we'd do that in `Validate::validate_ip2()` it would be misleading that an IPv6 cidr-notation is invalid which is generally not, only in case of MySQL.

Reply:
Added a new parameter for this, so the checks can stay in the validate method.